app/helpers/and.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+import { andHelper } from 'ember-truth-helpers/helpers/and';
+
+export default Ember.Helper.helper(andHelper);
```

Comment:
@rwjblue Ok, I will work on this check. Do you have any clue how we can test this guard? Right now tests does not catch this incompatibility with ember < 1.13.